packages/web-app/src/utils/validators.ts

```
@@ -180,10 +177,16 @@ export function isDaoNameValid(
   provider: InfuraProvider,
   setError: (name: string, error: FieldError) => void,
   clearError: (name?: string | string[]) => void,
-  getValues: (payload?: string | string[]) => Object
+  getValues: (payload?: string | string[]) => Object,
+  currentDaoEnsName: string
```

Does this validate the DAO name or the ENS? Or both? 

As it stands, entering an empty name (i.e., entering something and then deleting it), will return the error for empty dao name instead of an error for empty ens domain. This is easily solved by separating the validators.

---

Hey, It's already separated and Using just for ensName Validation, Not sure why did you faced this issue